### PR TITLE
lightning: fix dpanic log when init session vars

### LINKED ISSUE #37266: lightning: set system var dpanic
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

Use tidb-lightning to import data.

### 2. What did you expect to see? (Required)

Import successfuly.

### 3. What did you see instead (Required)

Import successfuly but there are a lot of `DPANIC` logs.
```
[2022/08/22 15:19:52.768 +08:00] [DPANIC] [session.go:268] ["new session: failed to set system var"] [tableName=`tpcc`.`orders`] [error="[variable:1238]Variable 'lc_time_names' is a read only variable"] [key=lc_time_names] [stack="github.com/pingcap/tidb/br/pkg/lightning/backend/kv.newSession\n\t/root/tidb/br/pkg/lightning/backend/kv/session.go:268\ngithub.com/pingcap/tidb/br/pkg/lightning/backend/kv.NewTableKVDecoder\n\t/root/tidb/br/pkg/lightning/backend/kv/kv2sql.go:114\ngithub.com/pingcap/tidb/br/pkg/lightning/backend/local.NewDuplicateManager\n\t/root/tidb/br/pkg/lightning/backend/local/duplicate.go:419\ngithub.com/pingcap/tidb/br/pkg/lightning/backend/local.(*local).CollectRemoteDuplicateRows\n\t/root/tidb/br/pkg/lightning/backend/local/local.go:1652\ngithub.com/pingcap/tidb/br/pkg/lightning/backend.Backend.CollectRemoteDuplicateRows\n\t/root/tidb/br/pkg/lightning/backend/backend.go:390\ngithub.com/pingcap/tidb/br/pkg/lightning/restore.(*TableRestore).postProcess\n\t/root/tidb/br/pkg/lightning/restore/table_restore.go:778\ngithub.com/pingcap/tidb/br/pkg/lightning/restore.(*Controller).restoreTables.func7.1\n\t/root/tidb/br/pkg/lightning/restore/restore.go:1624"]
[2022/08/22 15:19:56.506 +08:00] [DPANIC] [session.go:268] ["new session: failed to set system var"] [tableName=`tpcc`.`history`] [error="[variable:1238]Variable 'lc_time_names' is a read only variable"] [key=lc_time_names] [stack="github.com/pingcap/tidb/br/pkg/lightning/backend/kv.newSession\n\t/root/tidb/br/pkg/lightning/backend/kv/session.go:268\ngithub.com/pingcap/tidb/br/pkg/lightning/backend/kv.NewTableKVDecoder\n\t/root/tidb/br/pkg/lightning/backend/kv/kv2sql.go:114\ngithub.com/pingcap/tidb/br/pkg/lightning/backend/local.NewDuplicateManager\n\t/root/tidb/br/pkg/lightning/backend/local/duplicate.go:419\ngithub.com/pingcap/tidb/br/pkg/lightning/backend/local.(*local).CollectLocalDuplicateRows\n\t/root/tidb/br/pkg/lightning/backend/local/local.go:1634\ngithub.com/pingcap/tidb/br/pkg/lightning/backend.Backend.CollectLocalDuplicateRows\n\t/root/tidb/br/pkg/lightning/backend/backend.go:386\ngithub.com/pingcap/tidb/br/pkg/lightning/restore.(*TableRestore).postProcess\n\t/root/tidb/br/pkg/lightning/restore/table_restore.go:760\ngithub.com/pingcap/tidb/br/pkg/lightning/restore.(*Controller).restoreTables.func7.1\n\t/root/tidb/br/pkg/lightning/restore/restore.go:1624"]
```

### 4. What is your TiDB version? (Required)

master

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: close #37266

Problem Summary:

### What is changed and how it works?
- skip read-only vars

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by buchuitoudegou — APPROVED** [review]

File: br/pkg/lightning/backend/kv/session.go [hunk 1]
```diff
@@ -264,6 +264,15 @@ func newSession(options *SessionOptions, logger log.Logger) *session {
...
 		for k, v := range options.SysVars {
+			// since 6.3(current master) tidb checks whether we can set a system variable
+			// lc_time_names is a read-only variable for now, but might be implemented later.
+			if sv := variable.GetSysVar(k); sv == nil {
+				logger.DPanic("unknown system var", zap.String("key", k))
+				continue
+			} else if sv.ReadOnly {
+				logger.Info("skip read-only variable", zap.String("key", k))
+				continue
+			}
```
Comment (gozssky) [inline br/pkg/lightning/backend/kv/session.go:275]: We'd better not set `lc_time_names`. Otherwise, there are still many unnecessary logs.

Comment (D3Hunter) [reply br/pkg/lightning/backend/kv/session.go:275]: as explained in comments, not sure whether tidb will support it in later release

**Comment by D3Hunter** [comment]
@gozssky ptal

File: br/pkg/lightning/backend/kv/session.go (hunk 1 above)
Comment (gozssky) [reply br/pkg/lightning/backend/kv/session.go:275]: Can we reduce duplicate logs? In my test, importing 7G data will print 304 line logs like this.

Comment (D3Hunter) [reply br/pkg/lightning/backend/kv/session.go:276]: changed to debug, see https://github.com/pingcap/tidb/pull/37301/commits/632455456bc19de09949b846300ef01bc7446f07

**Review by gozssky — APPROVED** [review]

**Comment by D3Hunter** [comment]
ci success 
https://ci.pingcap.net/blue/organizations/jenkins/br_ghpr_unit_and_integration_test/detail/br_ghpr_unit_and_integration_test/13010/pipeline

**Pull request merged**